apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-logs-sidebar.tsx

```diff
@@ -59,7 +51,7 @@ export const TestWorkflowLogsSidebar = ({ transactionId }: TestWorkflowLogsSideb
       ) : activityId ? (
         <ActivityPanel
           activityId={activityId}
-          onActivitySelect={() => {}}
```

Comment:
This was not implemented at all and the click on View Execution was triggering the workflow.

Reply:
Good catch, missed this during the smaller PR's split

Reply:
@LetItRock what if we just set the activityId when onActivity select happens? Because we want to override it here? And then not fetching logic needs to happen here on the test-workflow-logs-sidebar as you moved all of the logic to the activity panel itself.